doc/getting-started/getting-started.rst

```diff
@@ -154,7 +154,8 @@ To set up the ACRN build environment on the development computer:
 
    .. code-block:: bash
 
-      sudo pip3 install lxml xmlschema defusedxml
+      sudo pip3 install lxml xmlschema defusedxml tqbm
+      ln -s /usr/bin/python3 /usr/bin/python
```

Comment:
This likely requires `sudo` privileges. But more importantly, why do we need this? IIRC we always specify `python3` throughout our documentation.

Reply:
Because configurator need to build by widows and linux. On widows python is used and linux python3 is needed.
So on linux, link python3 to python.

Reply:
Can you show me where there is a risk that Python 2 is used? The top-level `Makefile` explicitly call `python3`.

Reply:
python misc/config_tools/configurator/packages/configurator/thirdLib/manager.py clean

Reply:
Fix it there then please. I think it's better to not change the dev machine defaults especially if we can easily fix that in our code.

Reply:
@junjiemao1 @NanlinXie Do we have a requirement that the configurator must build and run on Windows?

Reply:
> @junjiemao1 @NanlinXie Do we have a requirement that the configurator must build and run on Windows?

I don't think so. In fact the recommendation is to use it on Debian-based Linux, though it is actually portable and can be built on any OS.

Reply:
So I  will consider the  Debian-based Linux.

Reply:
Done.